_posts/2022-08-19-dev-sprint-attendees.md

```diff
@@ -0,0 +1,44 @@
+---
+author: DjangoCon US Organizers
+category: General
+date: 2022-08-19 13:00:00
+layout: post
+title: "Development Sprints: What are they & why should I attend?"
+---
+
+## What even IS a Development Sprint?
+*(In the ancient days of FLOSS...)*
+Sprints are most common around **F**ree/**L**ibre **O**pen **S**ource **S**oftware, or [FLOSS](https://en.m.wikipedia.org/wiki/Alternative_terms_for_free_software) projects. Most of these projects (like Django and Python) are maintained by a community of volunteers. These volunteers usually work together while spread around the world. Conferences presented rare occasions for these developers to work together in person. And thus were born: Development Sprints! In the context of a given conference or event, you'll often hear them reffered to as "the sprints" or just "sprints".
+
+Over time, these core developers realized sprints also provided them an opportunity to help find *new* potential contributors. And that's where YOU come in.
+
+## Why Attend a Development Sprint at DCUS 2022?
+As mentioned in the "[Why DjangoCon US](https://2022.djangocon.us/why-djangocon-us/)" document, development sprints give you an opportunity to work shoulder-to-shoulder with experts in the community. In addition, there are practical and personal benefits to attending the sprints.
+
+From a practical perspective, if your organization makes heavy use of Django or a related package, getting involved in it's maintenance will give you a better understanding of that package, as well as it's strengths & limitations. On the personal side, most people won't usually work on a project that's as widely used as Django or some of the other related packages. Getting involved in the maintenance of one of these packages can expose you to engineering practices you like wouldn't see otherwise. This can help you become a better engineer, while allowing you to contribute to the community that benefits you and your company.
```

Comment:
```suggestion
From a practical perspective, if your organization makes heavy use of Django or a related package, getting involved in its maintenance will give you a better understanding of that package, as well as it's strengths & limitations. On the personal side, most people won't usually work on a project that's as widely used as Django or some of the other related packages. Getting involved in the maintenance of one of these packages can expose you to engineering practices you like wouldn't see otherwise. This can help you become a better engineer while allowing you to contribute to the community that benefits you and your company.
```

Couple minor grammar tweaks